log/stat/stat.go

```diff
@@ -0,0 +1,112 @@
+package stat
+
+import (
+	"bytes"
+	"fmt"
+	"strings"
+	"sync"
+	"text/tabwriter"
+
+	"github.com/peak/s5cmd/strutil"
+)
+
+const (
+	totalCount = iota
+	succCount
+)
+
+var (
+	enabled bool
+	stats   statistics
+)
+
+type statistics [2]syncMap
```

Comment:
why 2?

Reply:
one is for success count and the other for total count